roadmap/2022/MLOpsRoadmap2022.md

```diff
@@ -547,6 +557,16 @@ Solutions should provide the ability to specify guardrail metrics and facilitate
     <td>Government regulation of AI</td>
     <td>Proposed regulation creates an urgent requirement for the implementation of many of the technology requirements outlined in the roadmap as existing approaches will cease to be viable upon introduction. This in turn also introduces the need to provide traceable reporting to third parties whilst protecting intellectual property as part of the daily function of regulatory compliance.</td>  
   </tr>
+
+  <tr> 
+   <td>Providing mechanisms to treat ML models as part of the broader product/system in which they reside, rather than as independent products.</td>
+   <td>Tooling should enable models (and their supporting assets) to be distributed as reusable components within a larger system/application (similar to dependency management for software).<br/><br/>
+Tooling should enable explicit dependency management between:<br/>

```

Comment:
Revise to align to the product management view in the challenge. Perhaps reference the problem of asset management when making operational changes to live data rather than deploying through unified governance process?

Reply:
updated